### PR TITLE
fix installation issue on windows

### PR DESCRIPTION
**Pull request by tzing**
install from pip would failed on my windows
```
(tmpenv) λ pip install rope
Collecting rope
  Downloading rope-0.10.5.tar.gz (243kB)
    100% |████████████████████████████████| 245kB 2.1MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\tzu-ting\AppData\Local\Temp\pip-build-j1x3c_at\rope\setup.py", line 67, in <module>
        long_description=get_long_description(),
      File "C:\Users\tzu-ting\AppData\Local\Temp\pip-build-j1x3c_at\rope\setup.py", line 60, in get_long_description
        lines = open('README.rst').read().splitlines(False)
    UnicodeDecodeError: 'cp950' codec can't decode byte 0xc4 in position 407: illegal multibyte sequence

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\tzu-ting\AppData\Local\Temp\pip-build-j1x3c_at\rope\
```

just add `encoding='utf8'` to the line and it works

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 def get_long_description():
-    lines = open('README.rst', encoding='utf8').read().splitlines(False)
```
Comment (mcepl) [inline setup.py]: I usually prefer to use ``io.open`` which is meant to be used as the equivalent of py3k ``open``.

**Comment by mcepl** [comment]
OK, @abingham if you fix the conflicts soon enough, I am going to merge this on next Monday (February 10), unless @emacsway says otherwise.

**Comment by abingham** [comment]
@mcepl You sure you mean me and not @tzing?

**Comment by mcepl** [comment]
Sorry, yes, I meant @tzing , of course.

**Comment by tzing** [comment]
Oops I've delete the repo where I created this PR.
The conflict is solved on my newly created branch [issue-210](https://github.com/tzing/rope/tree/issue-210) but I don't know how to merge it into this PR again.

**Comment by mcepl** [comment]
> Oops I've delete the repo where I created this PR.
> The conflict is solved on my newly created branch [issue-210](https://github.com/tzing/rope/tree/issue-210) but I don't know how to merge it into this PR again.

Just open a new one.

**Pull request closed without merging**